src/component/Todos.js

```diff
@@ -0,0 +1,175 @@
+import React, { useEffect, useState } from "react";
+import Navbar from "./Navbar";
+import { AiOutlinePlusCircle } from "react-icons/ai";
+import TodoItem from "./TodoItem";
+import moment from "moment";
+
+const Todos = () => {
+  const [todoList, setTodoList] = useState([]);
+  const [isModal, setIsModal] = useState(false);
+  const [error, setError] = useState(false);
+  const [editTodoId, setEditTodoId] = useState(null);
+  const [currentTimeAndDate, setCurrentTimeAndDate] = useState(
+    moment().format("YYYY-MM-DDTHH:mm")
+  );
+  const [todoInput, setTodoInput] = useState({
+    todoTitle: "",
+    time: currentTimeAndDate,
+  });
+  const currentDateAndTime = moment().format("YYYY-MM-DDTHH:mm"); //this is use for input datetime minimum value
```

Comment:
if this is for `input datetime minimum value` update the variable name accordingly @amitark7